main/main.c

```diff
@@ -2153,6 +2153,26 @@ int php_module_startup(sapi_module_struct *sf, zend_module_entry *additional_mod
 #endif
 	REGISTER_MAIN_LONG_CONSTANT("PHP_DEBUG", PHP_DEBUG, CONST_PERSISTENT | CONST_CS);
 	REGISTER_MAIN_STRINGL_CONSTANT("PHP_OS", php_os, strlen(php_os), CONST_PERSISTENT | CONST_CS);
+#if defined(PHP_WIN32)
+	REGISTER_MAIN_BOOL_CONSTANT("PHP_WIN32", 1, CONST_PERSISTENT | CONST_CS);
```

Comment:
im thinking about naming it PHP_OS_WIN instead of PHP_WIN, what you think about this?